CC_DV_R_bm3024_F.Rmd

```diff
@@ -0,0 +1,200 @@
+---
+title: "Creating interactive plot by Plotly"
+author: "Brian Mao"
+date: "3/8/2021"
+output: html_document
+---
+
```

Comment:
Please remove lines 1-6. Follow 2.2.2 (https://jtr13.github.io/cc21/github-submission-instructions.html#submission-steps) for the same.